apps/docs/src/content/docs/reference/keep-latest.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: KeepLatest task
+description: the KeepLatest task function
+---
+
+import { LinkCard, Tabs, TabItem, Aside } from '@astrojs/starlight/components';
+
+This is the way to create a keepLatest task.
+
+This will run the initial tasks and then ensure that the very last task instance is also run. You
```

Comment:
I had a super quick look at the rest of the files since the PR is still open, and I don't understand what `keep-latest` is.

Reply:
Ah ok, I think you should explain it exactly this way, that's more clear 👍